cross/lz4/Makefile

```diff
@@ -13,5 +13,10 @@ COMMENT  = LZ4 is lossless compression algorithm, providing compression speed at
 LICENSE  = BSD
 
 CONFIGURE_TARGET = nop
+INSTALL_TARGET = lz4_install
 
 include ../../mk/spksrc.cross-cc.mk
+
+.PHONY: lz4_install
+lz4_install:
+	$(RUN) $(MAKE) DESTDIR=$(STAGING_INSTALL_PREFIX) install PREFIX=$(INTALL_DIR)
```

Comment:
I expect default install target should work https://github.com/SynoCommunity/spksrc/blob/master/mk/spksrc.install.mk#L59
Or else lz4 makefile does not follow DESTDIR GNU convention for makefile... It may worth to request fix in upstream code

Reply:
I have reverted this change as kiwix does not depend on cross/lz4 (and moved lz4 update to an update for borgbackup).